### ISSUE TITLE
The last modified time of parent directory always be 1970-01-01 08:00

### ISSUE THREAD

**Issue by huntersman** [s3fs-fuse/s3fs-fuse#2048]
#### Version of s3fs being used (s3fs --version)
1.91
#### Version of fuse being used (pkg-config --modversion fuse, rpm -qi fuse, dpkg -s fuse)
2.9.2
#### Kernel information (uname -r)
5.4.213-1.el7.elrepo.x86_64
#### GNU/Linux Distribution, if applicable (cat /etc/os-release)
CentOS7
#### s3fs command line used, if applicable
```
s3fs myBucket /mount/nfs -o passwd_file=${HOME}/.passwd-s3fs -o url=http://myIp -o use_path_request_style -o noxmlns -o dbglevel=error -o default_acl=public-read -o logfile=/var/log/s3fs.log -o allow_other
```
### Details about issue
The last modified time of parent directory doesn't change and always be 1970-01-01 08:00.

**Comment by huntersman** [comment]
Does not support by self maintained object storage.

**Comment by ggtakec** [comment]
@huntersman I'm sorry for late reply.
This specification has existed since the early days of s3fs.
However, the next version, if you mount with s3fs and update the stat information of the mount point, the stat information will be updated.
This fix was made in #1964. (PR #2016 under review is also relevant.)
If you want to try it now, you can build the code on the master branch.
Thanks for your help.
